.idea/description.html

```diff
@@ -0,0 +1 @@
+<html>Simple <b>Java</b> application that includes a class with <code>main()</code> method</html>
```

Comment:
As it has been already mentioned, you don't need files from .idea repository here. Remove them all.